install_config/router/default_haproxy_router.adoc

```diff
@@ -98,19 +98,21 @@ not exist, run the following to create a router:
 
 ifdef::openshift-enterprise[]
 ----
-$ oc adm router <router_name> --replicas=<number> --service-account=router
+$ oc adm router <router_name> --replicas=<number> --service-account=router --extended-logging=true
 ----
 endif::[]
 ifdef::openshift-origin[]
 ----
-$ oc adm router <router_name> --replicas=<number> --service-account=router
+$ oc adm router <router_name> --replicas=<number> --service-account=router --extended-logging=true
 ----
 endif::[]
 
 `--replicas` is usually `1` unless a
 xref:../../admin_guide/high_availability.adoc#[high availability] configuration
 is  being created.
 
+`--extended-logging=true` configures the router to forward access logs to your EFK stack.
```

Comment:
---> `--extended-logging=true`  enable a cartridge container to forward access logs to its standard output.

Reply:
@lihongan  Could you confirm that is a support feature or a workaround?

Reply:
I don't think so, I just create another sidecar container that running rsyslogd.

Reply:
It is not related to `EFK stack`,  should be `configures the router to forward HAProxy log to the syslog container.`